getting_started/setup_vm/roles/pebble/tasks/install.yml

```diff
@@ -0,0 +1,20 @@
+- name: Include vars
+  include_vars: common.yml
+
+- name: Create directory for pebble
+  file:
+    path: "{{ pebble_install_prefix }}"
+    state: directory
+  become: true
+
+- name: Download pebble
+  get_url:
+    url: https://github.com/letsencrypt/pebble/releases/{{ pebble_version }}/download/pebble_linux-amd64
```

Comment:
That's right, but this case is slightly different. When Let's Encrypt makes any changes to the protocol, we want to know about that as soon as possible, so that we can act before it reaches end-users. That's also why this is just in the dev/CI image, it shouldn't be installed in any other images.

Reply:
I imagine that LE don't update their protocol and break compatibility often (ever?) so I think it's OK to hardcode a version here and remember to bump it every now and then.

Reply:
Sure, it won't happen often, but if it does happen, we want to catch it. And nobody's going to remember to bump the version! Also, today's service already deviates from the RFC in two minor details and if that changes (for better or worse), we will have to adjust too.

Reply:
I couldn't find anything that would do this with minimal setup. We could probably check easily that the version of the dependency installed matches the version of the `latest` version on the dependencies GitHub repo, but it's definitely out of scope for this PR!